core/db_models/fields/EE_Email_Field.php

```diff
@@ -42,8 +43,8 @@ public function __construct($table_column, $nice_name, $nullable, $default_value
     public function prepare_for_set($email_address)
     {
         try {
-            $email_address = EmailAddressFactory::create($email_address);
-            return $email_address->get();
+            $email_address_obj = EmailAddressFactory::create($email_address);
+            return $email_address_obj instanceof EmailAddress ? $email_address_obj->get() : '';
```

Comment:
Golden Rule: if it can be null, it will be null!

Reply:
Yup, yup, yup.